tests/test_ws.py

```diff
@@ -0,0 +1,263 @@
+from starlette.testclient import TestClient
+from starlette.websockets import WebSocket  # Added import
+from quickhtml.routing import QuickHTMLWSEndpoint
+from quickhtml import QuickHTML
+
+
+def test_ws():
+    test_app = QuickHTML()
```

Comment:
Not needed within this PR, but we probably want this to become a fixture in the near future to make things easier